with-content-scripts-ui/contents/plasmo-mount.tsx

```diff
@@ -0,0 +1,21 @@
+import type { PlasmoContentScript } from "plasmo"
+
+export const config: PlasmoContentScript = {
+  matches: ["https://www.plasmo.com/*"]
+}
+
+export const getMountPoint = async () => document.querySelector("#pricing")
```

Comment:
I think it might be more intuitive if this were a function that takes in a Document and must return a single Node. So something like:

```
export const getMountPoint = async (dom: Document): Node => 
```

Reply:
I don't think so, the idea is that devs are at "root" - i.e they can use browser API here, nothing special. Introducing any parameter seems unintuitive to me - we want to leverage what people already know.